board/management/commands/getgamefromapi.py

```diff
@@ -38,33 +38,25 @@ def handle(self, *args, **options):
             for i in range(len(game.screen_url)):
                 Image.objects.update_or_create(url=game.screen_url[i], defaults={'game': g1})
 
-            platforms = Platforms.objects.filter(name__in=game.platforms)
+            platforms = Platform.objects.filter(name__in=game.platforms)
             genres = Genre.objects.filter(name__in=game.genres)
-            for platform in platforms:
-                g1.platforms.add(platform)
-            for genre in genres:
-                g1.genres.add(genre)
+            g1.platforms.add(*platforms)
+            g1.genres.add(*genres)
 
         else:
-            raise CommandError('There are no game with such parameters')
+            raise CommandError('There is no game with such parameters')
 
         self.stdout.write(self.style.SUCCESS(f'Successfully added game with id: {game_id}'))
 
-    @staticmethod
-    def _download_img(url):
-        response = get(url)
-        response.raise_for_status()
-        return response.content
-
     @staticmethod
     def _base_init():
-        platforms = Platforms.objects.all()
-        if platforms:
-            return None
-        params = {
-            'fields': 'name'
-        }
-        platforms = igdb_wrapper.get_platforms(params)
-        genres = igdb_wrapper.get_genres(params)
-        [Platforms.objects.create(id=item['id'], name=item['name']) for item in platforms]
-        [Genre.objects.create(id=item['id'], name=item['name']) for item in genres]
+        params = {'fields': 'name'}
+        platforms_api = igdb_wrapper.get_platforms(params)
+        genres_api = igdb_wrapper.get_genres(params)
+
+        platforms = Platform.objects.all()
+        genres = Genre.objects.all()
+        if platforms.count() != len(platforms_api):
+            [Platform.objects.create(id=item['id'], name=item['name']) for item in platforms]
+        if genres.count() != len(genres_api):
+            [Genre.objects.create(id=item['id'], name=item['name']) for item in genres]
```

Comment:
You should use regular `for` loops here instead of list comprehensions since the created list is not used anywhere.

Reply:
Also you should check for some unique parameter instead of just checking the count vs length since some platforms may change in `IGDB` at some point. Also if you have downloaded only 1 platform or genre for some reason (IGDB outage or smth) you'll end up with duplicates.